service/javadsl/api/src/main/java/com/lightbend/lagom/javadsl/api/Descriptor.java

```diff
@@ -390,14 +390,14 @@ public String toString() {
     /**
      * Describes a topic call.
      */
-    public static final class TopicCall<Message> {
+    public static final class TopicCall<MESSAGE> {
```

Comment:
There was a warning that referred to `Message` and I was being driven bonkers trying to figure out where the Message class was defined. Turns out it was a type parameter...

The convention in Java is all caps. I think this is the first time I've ever seen a type parameter in Jana code that wasn't in all caps, so it really caused a lot of confusion for me.